daemon/create.go

```diff
@@ -14,26 +14,35 @@ import (
 	"github.com/opencontainers/runc/libcontainer/label"
 )
 
+// ContainerCreate is the parameter set to ContainerCreate()
+type ContainerCreate struct {
+	Name            string
+	Config          *runconfig.Config
+	HostConfig      *runconfig.HostConfig
+	AdjustCPUShares bool
+	FromBuilder     bool
```

Comment:
@cpuguy83 - Fixed. Also rebased after the builder rework from @tiborvass which got merged into master. You can see the final update to address your above comments separately at https://github.com/Microsoft/docker/commit/c2e8745e6f70050baa215dad64a1ea920db5395b